crates/libcontainer/src/container/builder.rs

```diff
@@ -24,6 +25,12 @@ pub struct ContainerBuilder {
     /// The function that actually runs on the container init process. Default
     /// is to execute the specified command in the oci spec.
     pub(super) executor: Box<dyn Executor>,
+    // RawFd set to stdin of the container init process.
+    pub stdin: Option<RawFd>,
```

Comment:
I think it would be better for this to be `OwnedFd`.
With the current design as used in the doc example, this is leaking FDs.

Reply:
I also hesitated between OwnedFd and RawFd, At last I think maybe RawFd is better because with `RawFd` the caller has its control over when to close them, otherwise the fd is closed when the it go out of some scope, maybe it is not our expected.

Reply:
With `RawFd` there's a risk the caller closes the file before we use it.
If they dont want their file closed, and want to retain ownership, the caller can always call `try_clone()` to obtain a new `OwnedFd`.

Reply:
I think you are right, OwnedFd is good, Now The stdin/stdout/stderr in `ContainerBuilder` is `OwnedFd`, but in `ContainerArgs` it is still RawFd because it should implement `Clone` but `OwnedFd` does not implement `Clone`. So I changed it to `RawFd` with `as_raw_fd()` when new `ContainerArgs`, I'm not sure if that is appropriate, please take a look. Thanks